example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+          margin="dense"
+          onChange={(e)=>setWidth(e.target.value===''?'':Number(e.target.value))}
+          onBlur={()=>{
+            if (width < 0) {
+              setWidth(0);
+            } else if (width > 4000) {
+              setWidth(4000);
+            }
+          }}
+          inputProps={{
+            step: 1,
+            min: 0,
+            max: 4000,
+            type: 'number',
+            'aria-labelledby': 'input-slider',
+          }}
+        /> possible values 0 - 4000
+</Grid>
+<Grid item>
+height :  <Input
+style={{marginRight:10}}
+          value={height}
+          margin="dense"
+          onChange={(e)=>setHeight(e.target.value===''?'':Number(e.target.value))}
+          onBlur={()=>{
+            if (height < 0) {
+              setHeight(0);
+            } else if (height > 4000) {
+              setHeight(4000);
+            }
+          }}
+          inputProps={{
+            step: 1,
+            min: 0,
+            max: 4000,
+            type: 'number',
+            'aria-labelledby': 'input-slider',
+          }}
+        /> possible values 0 - 4000
+</Grid>
+<Grid item>
+quality :  <Input
+style={{marginRight:10}}
+          value={quality}
+          margin="dense"
+          onChange={(e)=>setQuality(e.target.value===''?'':Number(e.target.value))}
+          onBlur={()=>{
+            if (quality < 0) {
+              setQuality(0);
+            } else if (quality > 100) {
+              setQuality(100);
+            }
+          }}
+          inputProps={{
+            step: 1,
+            min: 0,
+            max: 100,
+            type: 'number',
+            'aria-labelledby': 'input-slider',
+          }}
+        /> possible values 0 - 100
+</Grid>
+<Grid item>
+background : <input style={{marginRight:10}} type="color" onChange={(e)=>setBackground(e.target.value.replace("#",""))}/>
+ only works for png with transparent background
+</Grid>
+<Grid item >
+output : <select onChange={(e)=>setOutput(e.target.value)}>
+  <option value="jpeg">jpeg</option>
+  <option value="jpg">jpg</option>
+  <option value="png">png</option>
+  <option value="gif">gif</option>
+  <option value="webp">webp</option>
+</select>
+</Grid>
+</Grid>
+<Grid item style={{marginTop:15}}>
+<PreviewImage appwrite={props.appwrite} width={width} height={height} quality={quality} background={background} output={output} id={props.imageId} />
+</Grid>
+
+   </div>
+  
+ )
+  
+};
+
+export { PreviewAndCrop };
+```
+example usage
+
+```js
+<PreviewAndCrop imageId={imageId} appwrite={appwrite} />
+```
+
+there is two props here:
+
+- ```imageId``` to get the state of imageId that we get from ```ListImages``` component and preview it using ```PreviewImage``` component
+- ```appwrite``` for using the appwrite SDK
+
+## Step 7 - Putting it all together in App.js
+
+So in order for this application to work we should put all of our component into ```App.js``` so that we can see how our cropped image looks like
```

Comment:
```suggestion
So in order for this application to work we should put all of our components into ```App.js``` so that we can see how our cropped image looks
```